### PR TITLE
kubernetes: update response to reflect AdmissionReview metadata

### PR DESCRIPTION
**Pull request by asadali**
addresses #34 
nit: i had to use `else` to ensure safe defaults.

### CODE REVIEW

File: kubernetes/admission/test_loadbalancer.rego [hunk 1]
```diff
@@ -3,34 +3,47 @@ package library.kubernetes.admission.test_loadbalancer
 import data.library.kubernetes.admission.example_loadbalancer as loadbalancer
 import data.library.kubernetes.admission.input.service
 
+v1beta1 := "admission.k8s.io/v1beta1"
+
+v1 := "admission.k8s.io/v1"
+
 test_clusterip {
-	# i := service.gen_input("ClusterIP", {}, null, null)
+	# i := service.gen_input("ClusterIP", {}, null, null, v1beta1)
```
Comment (patrick-east) [inline kubernetes/admission/test_loadbalancer.rego:11]: Should we just remove the commented out line here?

Comment (asadali) [reply kubernetes/admission/test_loadbalancer.rego:11]: no opinions. i couldn't figure out the original intent.

**Comment by tsandall** [comment]
LGTM

**Pull request merged**